src/inc/corjit.h

```diff
@@ -301,7 +301,6 @@ enum CorJitFuncKind
     CORJIT_FUNC_FILTER         // a funclet associated with an EH filter
 };
 
-#if !defined(FEATURE_USE_ASM_GC_WRITE_BARRIERS) && defined(FEATURE_COUNT_GC_WRITE_BARRIERS)
 // We have a performance-investigation mode (defined by the FEATURE settings above) in which the
```

Comment:
The comment below needs to be updated. You should probably mention the #defines above in the comment, since they are still used to build the VM and JIT.